test/models/internship_offers/api_test.rb

```diff
@@ -18,6 +18,7 @@ class ApiTest < ActiveSupport::TestCase
         zipcode: '60580',
         city: 'Coye la foret',
         sector: create(:sector),
+        school_type: InternshipOffer.school_types.keys.sample.to_sym,
```

Comment:
```suggestion
    school_type { InternshipOffer.school_types.keys.first.to_sym }
```
eviter le random sur les tests, si derrieredes comportements changent on va galerer a comprendre prk certains tests cassent...